index.js

```diff
@@ -1,12 +1,12 @@
 var createServer = require('http').createServer
-var resolve = require('url').resolve
+const url = require('url')
```

Comment:
```suggestion
const URL = require('url').URL
```

like the previous line.

Reply:
I was not sure whether to commit these changes or not due to the problems of istanbul.